CHANGELOG.md

```diff
@@ -7,12 +7,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.6.0] - 2021-06-07
```

Comment:
So, if no, unrelease section is present here we add it like below:

## [Unreleased]

### Fixed

### Added

### Changed

### Removed

While releasing the [release script](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/scripts/release.js#L103) updates this [Unreleased] to specific version and date, hence you need not add the version and date here.

Reply:
Thanks for the explanation, will update it